sdk/python/kfp/compiler/test_data/pipelines/two_step_pipeline_containerized.yaml

```diff
@@ -74,4 +74,4 @@ root:
       text:
         parameterType: STRING
 schemaVersion: 2.1.0
-sdkVersion: kfp-2.0.0-beta.2
+sdkVersion: kfp-2.0.0-beta.3
```

Comment:
This shouldn't matter. Our unit test when compare the compilation result with the golden file ignores the version. 
Do you hit any specific issue?

Reply:
Oh yes, I should close this PR instead. I was about to fix this two step pipeline test but realized that it's fixed in previous PRs already. Sorry for the mistake!